nav2_bringup/bringup/README.md

```diff
@@ -1,21 +1,21 @@
 # nav2_bringup
 
-The `nav2_bringup` package is an example bringup system for navigation2 applications.
+The `nav2_bringup` package is an example bringup system for Navigation2 applications.
```

Comment:
Is this a proper noun now ;-) I love having been a part of a proper noun

Reply:
:smiley:  It can be a proper noun if we use it everywhere in the stack.

I think we should try to use the same name in all documentation.

I have seen several different names in the repo. Some of them are;

- navigation2
- Navigation2
- Navigation 2 Stack
- ROS 2 Navigation 2
- ROS 2 navigation 2 stack
- ROS 2 Navigation System